src/components/Lists/ListSettings.vue

```diff
@@ -56,6 +56,23 @@
             </button>
         </section>
 
+        <section v-if="hasMultipleGames">
+            <h4>List Width</h4>
```

Comment:
Can you add this as a translation string? Recently finished adding all hard coded strings to i18n

Reply:
See #107 and #108

Reply:
Sure thing, I'm on my way out the door right now so I won't have it done for a while but it looks like there are actually a number of hard coded strings still on the page that I'll whack.

Reply:
Cheers buddy. I thought I caught them all, might be worth merging the latest master into your branch.